#!/usr/bin/env python
PACKAGE = "whs_navigation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("laptop_battery_limit", double_t, 0,
        "Shut down robot when laptop battery falls below this limit",
        20., 5., 100.)
gen.add("robot_battery_low", bool_t, 0, \
        "Simulate that the robot-battery is low",
        False)

exit(gen.generate(PACKAGE, "smach_patrol", "Parameters"))
